app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,60 @@
   </head>
 
   <body>
-    <%= yield %>
+
+ <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+
+    <a class="navbar-brand" href="http://nussucommit.com">
+    <img src="http://i64.tinypic.com/24cuyr7.png" width="150" height="70" alt="nussucommit">
+    </a>
+
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbar-content">
+    <ul class="navbar-nav">
+      
+        <li class="nav-item ">
+        <a class="nav-link" href="http://nussucommit.com">Home <span class="sr-only">(current)</span></a>
+        </li>
+      
+       	<li class="nav-item">
+        <a class="nav-link" href="#########">Grab Duty</a>
```

Comment:
Some of these have the correct links already, please link it properly :)

You can check the routes by using `bin/rails routes` in terminal, or just open something like `localhost:3000/asdf` lol